docker/iml-api.dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM rust-iml-base as builder
+
+FROM ubuntu
+COPY --from=builder /build/target/release/iml-api /usr/local/bin
+
+RUN apt-get update \
+    && apt-get install -y curl
+
+COPY docker/wait-for-dependencies.sh /usr/local/bin/
+ENTRYPOINT [ "wait-for-dependencies.sh" ]
```

Comment:
I didn't see it in here yet but I figure this service will eventually use postgres. If that's the case should we instead change this to:
```
ENTRYPOINT [ "wait-for-dependencies-postgres.sh" ]
```
?

Reply:
I think we can wait until we introduce a endpoint that depends on postgres.